node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -157,11 +157,13 @@ impl ChainScraper {
 
 		let block_hashes = std::iter::once(activated.hash).chain(ancestors);
 
+		let mut included_receipts: Vec<CandidateReceipt> = Vec::new();
 		let mut on_chain_votes = Vec::new();
 		for (block_number, block_hash) in block_numbers.zip(block_hashes) {
 			gum::trace!(?block_number, ?block_hash, "In ancestor processing.");
 
-			self.process_candidate_events(sender, block_number, block_hash).await?;
+			let events_for_block = self.process_candidate_events(sender, block_number, block_hash).await?;
```

Comment:
That is a pretty misleading name. `newly_included_receipts` would capture better what it does. (Especially considering that `process_candidate_events` is not only processing inclusion events.